lighthouse-extension/gulpfile.js

```diff
@@ -149,13 +149,14 @@ gulp.task('watch', ['lint', 'browserify', 'html', 'copyReportScripts'], () => {
     'app/scripts/**/*.js',
     'app/images/**/*',
     'app/styles/**/*',
-    'app/_locales/**/*.json'
+    'app/_locales/**/*.json',
+    'node_modules/lighthouse-core/**/*.js'
```

Comment:
does this do what you want it to do? It's just watching the copied module...to watch and use changes in actual `lighthouse-core/` I think you're going to have to go up a directory and do some kind of `npm install` dance to update what's in `node_modules`